src/backend/c-lisp.py

```diff
@@ -300,6 +341,8 @@ def gen_call_expr(self, expr, res_sym):
             arg_syms.append(arg_sym)
             instr_list += self.gen_expr(arg, res_sym=arg_sym)
         name = expr[1]
+        if not name in self.function_types:
```

Comment:
`name not in self.function_types` is more pythonic

Reply:
Done